nebari/template/stages/05-kubernetes-keycloak/main.tf

```diff
@@ -1,4 +1,4 @@
-resource "random_password" "keycloak-qhub-bot-password" {
+resource "random_password" "keycloak-nebari-bot-password" {
```

Comment:
uhm... this will change Keycloak admin-level password, not user interactable but its good to flag